solidity/core/contracts/AbacusConnectionManager.sol

```diff
@@ -42,12 +36,6 @@ contract AbacusConnectionManager is IAbacusConnectionManager, Ownable {
      */
     event NewOutbox(address indexed outbox);
```

Comment:
would love for us to standardize event naming outside of this PR before we hit mainnet. We use NewFooBar and SetFooBar and should just choose one. I just filed https://github.com/abacus-network/abacus-monorepo/issues/560

Reply:
Yes please, agreed.